web/components/Home/Resin/Subtractor.tsx

```diff
@@ -1,36 +1,37 @@
 import React, { memo } from "react";
 import { useConfig, useCurrentStats } from "../../../utils/config";
 import { useServerTime } from "../../../utils/time";
-import { clampResin, getResinRecharge } from "../../../db/resins";
+import { clampResin, getResinRecharge, ResinCap } from "../../../db/resins";
 import { trackEvent } from "../../../utils/umami";
 import { Button, ButtonGroup } from "@chakra-ui/react";
 import { useHotkeys } from "react-hotkeys-hook";
 import { useIntl } from "react-intl";
 
-const resinUsages = [60, 40, 30, 20];
-
 const Subtractor = ({ current }: { current: number }) => {
```

Comment:
I think we can simply call it `SideButton` since the file is already contained in the resin calculator folder. I'm surprised I didn't think about this.